edu.gemini.lch.model/src/main/java/edu/gemini/lch/model/LaserNight.java

```diff
@@ -199,17 +194,15 @@ public String getSemester(int offset) {
 
     /**
      * Gets the semester with an offset to the current night.
-     * @param offset
-     * @return
      */
-    public static String getSemester(DateTime day, int offset) {
+    public static String getSemester(ZonedDateTime day, int offset) {
         return getSemester(day.plusMonths(offset * 6));
     }
 
-    public static String getSemester(DateTime day) {
-        if (day.getMonthOfYear() < 2) {
+    public static String getSemester(ZonedDateTime day) {
```

Comment:
Don't we have this calculation already in ocs2 model? Maybe it is not worth adding a dependency just for this

Reply:
I wouldn't be surprised, but I can't seem to immediately locate it.